presto-native-execution/presto_cpp/main/tests/PrestoExchangeSourceTest.cpp

```diff
@@ -184,7 +184,7 @@ class Producer {
       future = std::move(f);
     }
 
-    std::move(future).get();
+    std::move(future).get(std::chrono::microseconds(10'000));
```

Comment:
Why the timeout?

Reply:
Without the timeout the test may hang. The timeout allows for the test to fail with an error message.